Services/User/classes/class.ilObjUserFolderGUI.php

```diff
@@ -520,6 +520,7 @@ public function confirmactivateObject(): void
                     true,
                     $ilUser->getId()
                 );
+                $obj->setLoginAttempts(0);
```

Comment:
IMO we should check if the user was inactive. Otherwise, we reset login attempts of active users.

```php
            if ($obj instanceof \ilObjUser) {
                if (!$obj->getActive()) {
                    $obj->setLoginAttempts(0);
                }
                $obj->setActive(
                    true,
                    $ilUser->getId()
                );
                $obj->update();
            }
```

Reply:
Thank you very much @mjansenDatabay ! You are completely right and I didn't think far enough. Will fix and merge.